data/json/items/tool/misc.json

```diff
@@ -368,21 +368,6 @@
     "charges_per_use": 1,
     "use_action": [ "WATER_PURIFIER" ]
   },
-  {
-    "type": "GENERIC",
-    "id": "mind_splicer",
-    "symbol": "o",
-    "color": "white",
-    "name": { "str": "mind splicer kit" },
-    "description": "Surgical forceps, cables and a modified smartphone inside a small plastic pouch.  Assembled to steal the mind of some poor man, these are tools of the creepy high-tech sandman.",
-    "material": [ "plastic" ],
-    "price": 12500,
-    "price_postapoc": 500,
-    "use_action": [ "MIND_SPLICER" ],
```

Comment:
there is a use action in C++ that could also possibly be removed I'm not sure how obsoleting actions works. Also probably best to do separately but worth making a note.